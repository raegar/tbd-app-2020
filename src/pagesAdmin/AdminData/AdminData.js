import React from 'react';
import {Container, Row} from 'react-bootstrap';
import './AdminData.css';
import AdminNavBar from '../../componentsAdmin/AdminNavBar/AdminNavBar';
import SearchBar from '../../componentsAdmin/AdminSearchBar/AdminSearchBar';

function AdminData(){
	return(
		<Container>
			<Row>
				<div class="col-sm-2">
					<AdminNavBar/>
				</div>
				<div class="col-lg-6">
					<SearchBar/>
				</div>
			</Row>
		</Container>
	);
}

export default AdminData;