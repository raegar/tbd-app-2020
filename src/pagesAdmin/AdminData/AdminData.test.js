import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import AdminData from './AdminData';
import { BrowserRouter as Router} from 'react-router-dom';


//cleans up 
afterEach(cleanup);

it("Renders AdminData", () => {
	const { asFragment } = render(<Router><AdminData/></Router>);
	expect(asFragment()).toMatchSnapshot();
});