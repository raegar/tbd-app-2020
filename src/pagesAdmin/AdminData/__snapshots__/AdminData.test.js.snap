// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders AdminData 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="row"
    >
      <div
        class="col-sm-2"
      >
        <div
          class="navbar-container"
        >
          <a
            href="/AdminDashboard"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminDashboard"
              type="button"
            >
              Dashboard​
            </button>
          </a>
          <a
            href="/AdminData"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminData"
              type="button"
            >
              Data​
            </button>
          </a>
          <a
            href="/AdminGraph"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminGraph"
              type="button"
            >
              Gender​
            </button>
          </a>
          <a
            href="/AdminGraph"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminGraph"
              type="button"
            >
              Age​
            </button>
          </a>
          <a
            href="/AdminGraph"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminGraph"
              type="button"
            >
              Course​
            </button>
          </a>
          <a
            href="/AdminGraph"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminGraph"
              type="button"
            >
              Ethnicity​
            </button>
          </a>
          <a
            href="/AdminGraph"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminGraph"
              type="button"
            >
              Location​
            </button>
          </a>
          <a
            href="/AdminGraph"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminGraph"
              type="button"
            >
              Income​
            </button>
          </a>
          <a
            href="/AdminGraph"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminGraph"
              type="button"
            >
              Disabilities​
            </button>
          </a>
          <a
            href="/AdminGraph"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminGraph"
              type="button"
            >
              Support​
            </button>
          </a>
          <a
            href="/AdminGraph"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminGraph"
              type="button"
            >
              Education​
            </button>
          </a>
          <a
            href="/AdminGraph"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminGraph"
              type="button"
            >
              Employment​
            </button>
          </a>
          <a
            href="/AdminGraph"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminGraph"
              type="button"
            >
              Payment​
            </button>
          </a>
          <a
            href="/AdminLogIn"
          >
            <button
              class="admin-navbar-button"
              data-testid="ucpButton"
              href="/AdminLogIn"
              type="button"
            >
              LogIn​
            </button>
          </a>
        </div>
      </div>
      <div
        class="col-lg-6"
      >
        <div>
          <label
            for="header-search"
          >
            <span
              class="visually-hidden"
            >
              Search blog posts 
            </span>
          </label>
          <input
            id="header-search"
            name="s"
            placeholder="Search data"
            type="text"
          />
          <button
            class="searchbar-container"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-thumbs-up fa-w-16 "
              data-icon="thumbs-up"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M104 224H24c-13.255 0-24 10.745-24 24v240c0 13.255 10.745 24 24 24h80c13.255 0 24-10.745 24-24V248c0-13.255-10.745-24-24-24zM64 472c-13.255 0-24-10.745-24-24s10.745-24 24-24 24 10.745 24 24-10.745 24-24 24zM384 81.452c0 42.416-25.97 66.208-33.277 94.548h101.723c33.397 0 59.397 27.746 59.553 58.098.084 17.938-7.546 37.249-19.439 49.197l-.11.11c9.836 23.337 8.237 56.037-9.308 79.469 8.681 25.895-.069 57.704-16.382 74.757 4.298 17.598 2.244 32.575-6.148 44.632C440.202 511.587 389.616 512 346.839 512l-2.845-.001c-48.287-.017-87.806-17.598-119.56-31.725-15.957-7.099-36.821-15.887-52.651-16.178-6.54-.12-11.783-5.457-11.783-11.998v-213.77c0-3.2 1.282-6.271 3.558-8.521 39.614-39.144 56.648-80.587 89.117-113.111 14.804-14.832 20.188-37.236 25.393-58.902C282.515 39.293 291.817 0 312 0c24 0 72 8 72 81.452z"
                fill="currentColor"
              />
            </svg>
             Search
          </button>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
